### ISSUE TITLE
appveyor is confused by a project moving from a personal account to an organization

### ISSUE THREAD

**Issue by nico** [appveyor/ci#2237]
Hi,

I had set up appveyor for https://github.com/martine/ninja , and it had worked fine there.

A while ago, we converted the project to an organization; it now lives at https://github.com/ninja-build/ninja/ Ever since the move, appveyor still builds things for pull requests and issues, but no longer leaves any comment in the github web ui.

https://ci.appveyor.com/project/nico/ninja/settings has "github repository" set to "ninja-build/ninja". The webhook is installed in https://github.com/ninja-build/ninja/ (and builds do get triggered).

How can I get appveyor to push build results to https://github.com/ninja-build/ninja again?

**Comment by IlyaFinkelshteyn** [comment]
Probably GitHub user authorizing GitHub at https://ci.appveyor.com/account has no enough permissions to set statuses. Or if use has enough permission, OAuth token should be refreshed. In any case, try revoke GitHub authorization and re-authorize GitHub on that page. Let us know if this worked fro you.

**Comment by nico** [comment]
Actually: Builds now show up on pull requests, which is a huge step forward. See e.g. https://github.com/ninja-build/ninja/pull/1416 However, the build there is https://ci.appveyor.com/project/nico/ninja/build/1.0.747 which is already done (and failed), yet it still shows up as pending on the pull request. So the triggering works, but commenting when the build is done apparently doesn't, or is very delayed for some reason.

**Comment by nico** [comment]
Hm, now it shows up as failed. Apparently there's a tens-of-seconds window (at least) where the pull request isn't up-to-date.
